autofill/autofill-impl/src/test/java/com/duckduckgo/autofill/ui/credential/management/suggestion/SuggestionMatcherTest.kt

```diff
@@ -69,6 +70,13 @@ class SuggestionMatcherTest {
         assertEquals(2, suggestions.size)
     }
 
+    @Test
+    fun whenSubdomainIncludedInSavedSiteAndVisitingRootSiteThenSuggestionOffered() {
+        val creds = listOf(creds("https://duckduckgo.com"))
+        val suggestions = testee.getSuggestions("https://test.duckduckgo.com", creds)
+        assertEquals(1, suggestions.size)
```

Comment:
NIT: would be good to compare that the correct  suggestions are being returned than just checking the sise